<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LocationConditionDialog</class>
 <widget class="QDialog" name="LocationConditionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>733</width>
    <height>468</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Location condition</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="1" column="2" rowspan="4" colspan="2">
    <widget class="QGroupBox" name="stateGroup">
     <property name="title">
      <string>State(s)</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="ExtendedTableWidget" name="stateTable">
        <property name="columnTitles">
         <stringlist>
          <string>State</string>
          <string>Value</string>
         </stringlist>
        </property>
        <widget class="QFrame" name="container">
         <layout class="QFormLayout" name="formLayout_4">
          <item row="0" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>State:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="stateEdit"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Value:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="stateValueCombo">
            <property name="editable">
             <bool>true</bool>
            </property>
            <property name="insertPolicy">
             <enum>QComboBox::NoInsert</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QGroupBox" name="fluidGroup">
     <property name="title">
      <string>Fluid</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QFormLayout" name="formLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Fluid:</string>
        </property>
        <property name="buddy">
         <cstring>fluidSelector</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="IdTagSelector" name="fluidSelector"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Position</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="1">
       <widget class="NumberProvider" name="xInput">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>22</height>
         </size>
        </property>
        <property name="modes">
         <set>NumberProvider::ExactAndRange</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="NumberProvider" name="yInput">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>22</height>
         </size>
        </property>
        <property name="modes">
         <set>NumberProvider::ExactAndRange</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="NumberProvider" name="zInput">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>22</height>
         </size>
        </property>
        <property name="modes">
         <set>NumberProvider::ExactAndRange</set>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>X:</string>
        </property>
        <property name="buddy">
         <cstring>xInput</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>Y:</string>
        </property>
        <property name="buddy">
         <cstring>yInput</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>Z:</string>
        </property>
        <property name="buddy">
         <cstring>zInput</cstring>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="1" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="blockGroup">
     <property name="title">
      <string>Block</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>NBT:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="InventorySlot" name="blockSlot">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Block:</string>
        </property>
        <property name="buddy">
         <cstring>blockSlot</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPlainTextEdit" name="blockNbtEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="TrueFalseBox" name="smokeyCheck">
     <property name="text">
      <string>Smokey</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="TrueFalseBox" name="canSeeSkyCheck">
     <property name="text">
      <string>Can see the sky</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" colspan="2">
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Biome:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Dimension:</string>
       </property>
       <property name="buddy">
        <cstring>dimensionCombo</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="dimensionCombo">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>0</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>(not set)</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Structure:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="NumberProvider" name="lightInput">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>22</height>
        </size>
       </property>
       <property name="modes">
        <set>NumberProvider::ExactAndRange</set>
       </property>
       <property name="maxLimit">
        <double>15.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>Light:</string>
       </property>
       <property name="buddy">
        <cstring>lightInput</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="IdTagSelector" name="biomeSelector"/>
     </item>
     <item row="2" column="1">
      <widget class="IdTagSelector" name="structureSelector"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ExtendedTableWidget</class>
   <extends>QWidget</extends>
   <header>extendedtablewidget.h</header>
   <addpagemethod>setContainer</addpagemethod>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>NumberProvider</class>
   <extends>QFrame</extends>
   <header>numberprovider.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>IdTagSelector</class>
   <extends>QFrame</extends>
   <header>idtagselector.h</header>
  </customwidget>
  <customwidget>
   <class>TrueFalseBox</class>
   <extends>QCheckBox</extends>
   <header location="global">truefalsebox.h</header>
  </customwidget>
  <customwidget>
   <class>InventorySlot</class>
   <extends>QFrame</extends>
   <header>inventoryslot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>LocationConditionDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>259</x>
     <y>548</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>LocationConditionDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>327</x>
     <y>548</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
